examples/pyvista.py

```diff
@@ -0,0 +1,114 @@
+"""
+.. _add_example_example:
```

Comment:
This link is a bit confusing at first sight. Use the same name as the one for the filename.

Reply:
Also, please rename this file to `basic_example.py`. It is a good practice to avoid naming your files as Python modules. Consider:

- `pyvsita.py` -> `example_pyvista.py`
- `matplotlib.py` -> `example_matplotlib.py`